apophis/cmds/remind.py

```diff
@@ -0,0 +1,159 @@
+""" Reminder Command """
+from cmds.command import Command
+from datetime import datetime
+
+
+"""
+Time Conversion Function
+
+This method takes a given unit of time and converts it datetime format.
+Time unit example: 1 year(s), 1 month(s), 1 day(s), 1 hour(s), 1 minute(s)
+1 second(s)
+"""
+
+
+def convert_times(when):
+    """
+    As LiquidLemon pointed out in review, this can be reduced.
```

Comment:
This should be turned into an issue, and insert a link to it here

Reply:
yes ma'am